pkg/virt-operator/webhooks/webhook_test.go

```diff
@@ -83,9 +85,10 @@ var _ = Describe("Webhook", func() {
 		})
 
 		It("should deny the deletion if a VMIRS exists", func() {
+			_, err := fakeClient.KubevirtV1().VirtualMachineInstanceReplicaSets(k8sv1.NamespaceDefault).Create(context.TODO(), &v1.VirtualMachineInstanceReplicaSet{}, metav1.CreateOptions{})
```

Comment:
There is no need for name?

Reply:
Actually not, but for being closer to reality I can add it.
It wasn't there before.